plugins/modules/database/postgresql/postgresql_slot.py

```diff
@@ -70,6 +70,11 @@
     - Permissions checking for SQL commands is carried out as though
       the session_role were the one that had logged in originally.
     type: str
+  trust_input:
+    description:
+    - If C(no), check whether values of some parameters are potentially dangerous.
```

Comment:
```suggestion
    - If C(no), check whether a value of I(session_role) is potentially dangerous.
    - It does make sense to use C(yes) only when SQL injections via I(session_role) are possible.
```
Considering that only `session_role` will be checked, i suggest pointing this out explicitly.
And we could put this explanations into all next modules in such cases where the parameter can be used for checking a small number of their other parameters.
What do you think?

Reply:
Makes perfect sense to me, have updated the docs.